content/webapp/services/prismic/transformers/json-ld.ts

```diff
@@ -129,6 +130,30 @@ export function eventLd(event: Event): JsonLdObj[] {
     });
 }
 
+export function articleSeriesLd(series: Series): JsonLdObj {
+  return objToJsonLd(
+    {
+      headline: series.title,
+      contributor: series.contributors.map(({ contributor }) => {
+        const type = contributor.type === 'people' ? 'Person' : 'Organization';
+        return objToJsonLd(
+          {
+            name: contributor.name,
+            image: contributor.image
+              ? getImageUrlAtSize(contributor.image, { w: 600 })
+              : undefined,
+          },
+          { type, root: false }
+        );
+      }),
```

Comment:
We seem to create contributor LD quite a lot in this file; is it worth pulling it out into a standalone function?